src/Aspire.ProjectTemplates/templates/aspire-empty/.template.config/localize/templatestrings.es.json

```diff
@@ -8,7 +8,7 @@
   "symbols/AspireVersion/description": "La versión de .NET Aspire que se va a usar.",
   "symbols/AspireVersion/displayName": "Versión de .NET Aspire",
   "symbols/AspireVersion/choices/9.1/displayName": "9.1",
-  "symbols/AspireVersion/choices/9.1/description": "Chooses .NET Aspire 9.1",
+  "symbols/AspireVersion/choices/9.1/description": "Elige .NET Pm 9.1",
```

Comment:
```suggestion
  "symbols/AspireVersion/choices/9.1/description": "Elige .NET Aspire 9.1",
```

Reply:
added to https://ceapex.visualstudio.com/CEINTL/_queries/edit/1027023/?queryId=bffccee5-495f-4601-b154-c6f02aa9cf61

incidentally query to find such bugs https://ceapex.visualstudio.com/CEINTL/_queries/query-edit/bffccee5-495f-4601-b154-c6f02aa9cf61/